xchange-core/src/main/java/org/knowm/xchange/dto/account/Wallet.java

```diff
@@ -24,15 +20,24 @@
   private String id;
   /** A descriptive name for this wallet. Defaults to {@link #id} */
   private String name;
+  /** All wallet operations */
+  private final Set<WalletFeature> walletFeatures;
```

Comment:
rename attribute to `features` and getter to `getFeatures()`

further more I think this attribute should never be null
for constructors that don't have it in the args I think we can safely assume the following features: FUNDING, TRADING <- as this is the most common case

Reply:
Can i add a new "getter" in order to return the default features?

Reply:
I don't understand how a new getter helps with the problem of getFeatures() returning null in all the current modules